FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Common/Configurations/DicomServerConfiguration.cs

```diff
@@ -17,5 +17,8 @@ public class DicomServerConfiguration
 
         [JsonProperty("authentication")]
         public AuthenticationType Authentication { get; set; } = AuthenticationType.None;
+
+        [JsonProperty("serverEnvironmentGroup")]
+        public ServerEnvironmentGroup ServerEnvironmentGroup { get; set; } = ServerEnvironmentGroup.PROD;
```

Comment:
Maybe we can directly use Audience directly here?

Reply:
Sure, updated.